templates/azure/index.html

```diff
@@ -0,0 +1,174 @@
+            loading="auto",
+          ) | safe
+        }}
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <div class="col-12">
+      <h2 class="p-muted-heading u-align--center">UBUNTU ON AZURE POWERS CLOUD LEADERS LIKE</h2>
+      <ul class="p-inline-images u-no-margin--bottom">
+        <li class="p-inline-images__item">
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/b3315d88-walmart.svg",
+              alt="Walmart",
+              height=60,
+              width=200,
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-inline-images__logo"},
+            ) | safe
+          }}
+        </li>
+        <li class="p-inline-images__item">
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/0143bd92-AT%26T_logo_2016.svg",
+              alt="AT&amp;T",
+              height=60,
+              width=200,
+              hi_def=True,
+              loading="lazy",
+              attrs={"class": "p-inline-images__logo"},
+            ) | safe
+          }}
+        </li>
+      </ul>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="row">
+    <div class="col-6">
+      <h2>Choose the right Ubuntu for you</h2>
+      <p>Select between standard Ubuntu Server and the Pro edition, which includes expanded security and Azure integration:</p>
+    </div>
+  </div>
+
+  <div class="row u-equal-height">
+    <div class="col-6 p-card">
+      <h3 class="p-card__title">Ubuntu</h3>
+      <p class="p-heading--two">Free to use</p>
+      <hr class="u-sv1">
+      <div class="p-card__content">
+        <ul class="p-list">
+          <li class="p-list__item is-ticked">Azure-optimised kernel and userspace components built from the latest releases</li>
+          <li class="p-list__item is-ticked">Broad set of open source application components available for development</li>
+          <li class="p-list__item is-ticked">Security patches for the kernel and key infrastructure components</li>
+          <li class="p-list__item is-ticked">Updates mirrored and images published in all Azure regions worldwide</li>
+          <li class="p-list__item is-ticked">5-year lifetime</li>
+        </ul>
+        <p>Run Ubuntu 18.04 LTS: <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/Canonical.UbuntuServer?tab=Overview" class="p-link--external">Standard</a></p>
+      </div>
+    </div>
+    <div class="col-6 p-card">
+      <h3 class="p-card__title">Ubuntu Pro for Azure</h3>
+      <p class="p-heading--two">Starting at 0.1c/hour</p>
+      <hr class="u-sv1">
+      <div class="p-card__content">
+        <ul class="p-list">
+          <li class="p-list__item is-ticked">Includes all Ubuntu optimisations and updates of the free version</li>
+          <li class="p-list__item is-ticked">Expanded security covering application components delivered with Ubuntu</li>
```

Comment:
The copy doc has this item as "Expanded security covering 26,000 additional application components delivered with Ubuntu"